tests/test_dagmc_stats.py

```diff
@@ -104,39 +108,85 @@ def test_get_triangle_aspect_ratio(self):
-        # needs fixing !
-
-        #exp = (10*10*10*np.sqrt(2))/(8*5*np.sqrt(2)*5*np.sqrt(2)*(10-5*np.sqrt(2)))
-        #obs = dagmc_stats.get_triangle_aspect_ratio(my_core, root_set)[0]
-        #assertAlmostEqual(exp,obs)
-        #
-        #exp = my_core.get_entities_by_type(root_set, types.MBTRI).size()
-        #obs = len(dagmc_stats.get_triangle_aspect_ratio(my_core, root_set))
-        #assertEqual(exp,obs)
+        exp = (10*10*10*np.sqrt(2))/(8*5*np.sqrt(2)*5*np.sqrt(2)*(10-5*np.sqrt(2)))
+        obs = ds.get_triangle_aspect_ratio(my_core_2, root_set_2, dagmc_tags_2['geom_dim'])[0]
+        self.assertAlmostEqual(exp, obs)
+        
+        exp = my_core_2.get_entities_by_type(root_set_2, types.MBTRI).size()
+        obs = len(ds.get_triangle_aspect_ratio(my_core_2, root_set_2, dagmc_tags_2['geom_dim']))
+        self.assertEqual(exp, obs)
 
 
     def test_get_area_triangle(self):
         """
         Tests part of the get__area_triangle function
         """
-        test_input2 = "tests/single-cube.h5m"
-        my_core = core.Core()
-        my_core.load_file(test_input2)
-        root_set = my_core.get_root_set()
-
-        # !! Test is failing because functions are not called with proper arguments
-        # needs fixing !
-
-        #exp = 50
-        #obs = dagmc_stats.get_area_triangle(my_core, root_set)[0]
-        #assertEqual(exp,obs)
-        #
-        #exp = my_core.get_entities_by_type(root_set, types.MBTRI).size()
-        #obs = len(dagmc_stats.get_area_triangle(my_core, root_set))
-        #assertEqual(exp,obs)
+        exp = 50
+        obs = ds.get_area_triangle(my_core_2, root_set_2, dagmc_tags_2['geom_dim'])[0]
+        self.assertAlmostEqual(exp, obs)
+        
+        exp = my_core_2.get_entities_by_type(root_set_2, types.MBTRI).size()
+        obs = len(ds.get_area_triangle(my_core_2, root_set_2, dagmc_tags_2['geom_dim']))
+        self.assertEqual(exp, obs)
+
+
+    def test_get_tri_vert_data(self):
+        """Tests part of the get_tri_vert_data function
+        """
+        tri_vert_data, all_verts = ds.get_tri_vert_data(my_core_2, native_ranges_2)
+        
+        exp = 2*6*3
+        obs = len(tri_vert_data)
+        self.assertEqual(exp, obs)
+        
+        exp = 8
+        obs = len(all_verts)
+        self.assertEqual(exp, obs)
+        
+        exp = [5, 5, 4, 4, 5, 4, 5, 4]
+        for i in range(8):
+            obs = len(tri_vert_data[tri_vert_data['vert'] == all_verts[i]])
+            self.assertTrue(obs == exp[i])
+        
+        
+    def test_get_gaussian_curvature(self):
+        """Tests part of the get_gaussian_curvature function
+        """
+        tri_vert_data, all_verts = ds.get_tri_vert_data(my_core_2, native_ranges_2)
+        gc_all = ds.get_gaussian_curvature(my_core_2, native_ranges_2,all_verts, tri_vert_data)
+        
+        exp = 8
+        obs = len(gc_all)
+        self.assertEqual(exp, obs)
+        
+        exp = 0.5*np.pi
+        for i in range(8):
```

Comment:
numpy has an array for an approx comparison of two array: https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html#numpy.testing.assert_allclose